meta-luneos/recipes-webos/db8/db8.bb

```diff
@@ -53,8 +51,8 @@ do_install_append() {
     install -m 0644 ${S}/files/systemd/${PN}-tempdb.service ${D}${systemd_unitdir}/system/
 }
 
-PV = "3.2.0-5+git${SRCPV}"
-SRCREV = "4852de65c2a05cb25da385afedca73aaa246d215"
+PV = "3.2.0-1+git${SRCPV}"
```

Comment:
Please change to 3.2.0-5 as per https://github.com/webosose/meta-webosose/blob/master/meta-webos/recipes-webos/db8/db8.bb#L21

Reply:
oops, sorry, mistake during the merge. will fix.